README.md

```diff
@@ -238,13 +283,35 @@ The **summary.tsv** output file generated by `staramr` contains the following co
 | SRR1952908 | aadA1, aadA2, blaTEM-57, cmlA1, gyrA (S83Y), sul3, tet(A) | streptomycin, ampicillin, chloramphenicol, ciprofloxacin I/R, nalidixic acid, sulfisoxazole, tetracycline |
 | SRR1952926 | blaTEM-57, gyrA (S83Y), tet(A)                            | ampicillin, ciprofloxacin I/R, nalidixic acid, tetracycline                                               |
 
+## detailed_summary.tsv
+
+The **detailed_summary.tsv** output file generated by `staramr` contains the following columns:
+
+* __Isolate ID__: The id of the isolate/genome file(s) passed to `staramr`.
+* __Gene__: The particular AMR gene detected from ResFinder, PlasmidFinder, and PointFinder.
+* __Predicted Phenotype__: The predicted AMR phenotype (drug resistances) found in ResFinder.
+* __%Identity__: The % identity of the top BLAST HSP to the AMR gene.
```

Comment:
Remove mention of **AMR** here and in entries below since plasmid genes aren't AMR genes.

So, it would become:

>The % identity of the top BLAST HSP to the gene.

Reply:
Solved in commit 2202e45